sopel/bot.py

```diff
@@ -438,12 +447,15 @@ def reply(self, text, dest, reply_to, notice=False):
             self.say(text, dest)
 
     def call(self, func, sopel, trigger):
-        """Call a function, after checking whether the function is restricted or subject to limits.
-
-        :param function func: the function to call
-        :param SopelWrapper sopel: a SopelWrapper instance
-        :param Trigger trigger: the Trigger object for the line from the server that triggered
-            this call
+        """Call a function (``func``), after checking if the function is
+        restricted or subject to limits.
```

Comment:
As above, no need to point out the parameter name in the short description when we have param lists.

Would also really like to keep short descriptions to one line only. Can't make suggestions that apply to multiple lines, so apply this (and tweak it, if you have a better idea) however you may manually:

```
    """Call a function, applying any rate-limits or restrictions.
```

Reply:
Done!